### PR TITLE
BR: add a doc about Azure Blob Storage

### PR DESCRIPTION
**Pull request by en-jin19**
### What is changed, added or deleted? (Required)
added a doc about Azure Blob Storage

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- https://github.com/pingcap/docs-cn/pull/8063
- https://github.com/pingcap/docs-cn/pull/8155
- https://github.com/pingcap/docs-cn/pull/8194
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: br/backup-and-restore-azblob.md [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+summary: Learns how to use BR to backup and restore data on the external storage Azure Blob Storage
+---
+
+# Back up and Restore Data on Azure Blob Storage
+
+The Backup & Restore (BR) tool supports using Azure Blob Storage as the external storage for backing up and restoring data.
+
+For detailed information on other external storages supported by BR, refer to [External Storages](/br/backup-and-restore-storages.md).
+
+## Use cases
```
Comment (en-jin19) [inline br/backup-and-restore-azblob.md:12]:
```suggestion
## User scenario
```

File: br/backup-and-restore-storages.md [hunk 2]
```diff
@@ -128,11 +159,18 @@ If you have specified URL parameters and command-line parameters at the same tim
...
 | `--gcs.credentials-file` |  The path of the JSON-formatted credential on the tool node. |
 | `--gcs.storage-class` | The storage type of the upload object, such as `STANDARD` and `COLDLINE`.  |
 | `--gcs.predefined-acl` | The pre-defined ACL of the upload object, such as `private` and `project-private`. |
 
+### Azblob command-line parameters
+
+| Command-line parameter | Description |
+| `--azblob.account-name` | The account name of the storage |
+| `--azblob.account-key` | The access key |
+| `--azblob.access-tier` | Storage class of the uploaded objects (for example, `Hot`, `Cool`, `Archive`). If `access-tier` is not set (the value is empty), the value is `Hot` by default. |
```
Comment (Leavrth) [inline br/backup-and-restore-storages.md:172]: storage class
->
access tier

File: br/backup-and-restore-azblob.md [hunk 3]
```diff
@@ -0,0 +1,163 @@
...
+With BR, you can back up and restore data on Azure Blob Storage in the following two methods:
+
+- Backup and restore using Azure AD
+- Backup and restore using an access key
+
+In common cases, to avoid exposing the key information (such as `account-key`) in command lines, it is recommended to use the first method - use Azure AD.
+
+The following is an example of backup and restore operations on Azure Blob Storage using the above two methods. The purpose of the operations are as follows:
+
+- Back up: Back up the `test` library to a place with container name `container=test` and path prefix `t1` in Azure Blob Storage.
```
Comment (en-jin19) [inline br/backup-and-restore-azblob.md:27]:
```suggestion
- Back up: Back up the database named `test` to a place with container name `container=test` and path prefix `t1` in Azure Blob Storage.
```

File: br/backup-and-restore-azblob.md [hunk 4]
```diff
@@ -0,0 +1,163 @@
...
+- Restore: Restore the place with the container name `container=test` and the path prefix `t1` in Azure Blob Storage to the `test` library.
```
Comment (en-jin19) [inline br/backup-and-restore-azblob.md:28]:
```suggestion
- Restore: Restore the place with the container name `container=test` and the path prefix `t1` in Azure Blob Storage to the database named test.
```

File: br/backup-and-restore-azblob.md [hunk 5]
```diff
@@ -0,0 +1,163 @@
...
+
+#### Back up
+
+When backing up data using Azure AD, you need to specify `account-name` and `access-tier`. If `access-tier` is not set (the value is empty), the value is `Hot` by default.
+
+> **Note:**
+>
+> When using Azure Blob Storage as the external storage, you must set `send-credentials-to-tikv = true` (which is set by default). Otherwise, the backup task will fail.
+
+This section shows baking up data to `cool tier`, that is, the storage class of the uploaded object is `Cool`. You can specify `account-name` and `access-tier` in two ways:
```
Comment (en-jin19) [inline br/backup-and-restore-azblob.md:44]:
```suggestion
This section shows backing up data to `cool tier`, that is, the storage class of the uploaded object (`access-tier`) is `Cool`. You can specify `account-name` and `access-tier` in two ways:
```

File: br/backup-and-restore-azblob.md [hunk 6]
```diff
@@ -0,0 +1,163 @@
...
+
+    ```
+    tiup br restore db --db test -u 127.0.0.1:2379 -s 'azure://test/t1?' --azblob.account-name=devstoreaccount1
+    ```
+
+### Method 2: Back up and restore using an access key (easy)
+
+#### Back up
+
+When backing up data using an access key, you need to specify `account-name` and `access-tier`. If `access-tier` is not set (the value is empty), the value is `Hot` by default.
```
Comment (en-jin19) [inline br/backup-and-restore-azblob.md:78]:
```suggestion
When backing up data using an access key, you need to specify `account-name`, `account-key`, and `access-tier`. If `access-tier` is not set (the value is empty), the value is `Hot` by default.
```

File: br/backup-and-restore-azblob.md [hunk 7]
```diff
@@ -0,0 +1,163 @@
...
+
+> **Note:**
+>
+> When using Azure Blob Storage as the external storage, you must set `send-credentials-to-tikv = true` (which is set by default). Otherwise, the backup task will fail.
+
+This section shows baking up data to `cool tier`, that is, the storage class of the uploaded object is `Cool`. You can specify `account-name` and `access-tier` in two ways:
```
Comment (en-jin19) [inline br/backup-and-restore-azblob.md:84]:
```suggestion
This section shows backing up data to `cool tier`, that is, the storage class of the uploaded object (`account-tier`) is `Cool`. You can specify `account-name`, `account-key`, and `access-tier` in two ways:
```

File: br/backup-and-restore-azblob.md [hunk 8]
```diff
@@ -0,0 +1,163 @@
...
+
+- Write the parameters information in command-line parameters:
+
+    ```
+    tiup br backup db --db test -u 127.0.0.1:2379 -s 'azure://test/t1?' --azblob.account-name=devstoreaccount1 --azblob.account-key=Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw== --azblob.access-tier=Cool
+    ```
+
+#### Restore
+
+When restoring data using an access key, you need to specify `account-name` and `access-tier`. You can specify the parameters in two ways:
```
Comment (en-jin19) [inline br/backup-and-restore-azblob.md:100]:
```suggestion
When restoring data using an access key, you need to specify `account-name` and `account-key`. You can specify the parameters in two ways:
```

File: br/backup-and-restore-azblob.md [hunk 9]
```diff
@@ -0,0 +1,163 @@
...
+## Parameter specification
+
+During the backup and restore process, you need to use parameters `account-name`, `account-key`, `access-tier`. The following is the detailed description of the parameters:
+
+- [URL parameters](/br/backup-and-restore-storages.md#azblob-url-parameters)
+- [Command-line parameters](/br/backup-and-restore-storages.md#azblob-command-line-parameters)
+
+### Configure environment variables as parameters
+
+When baking up and restoring data using Azure AD, the environment variables `$AZURE_CLIENT_ID`, `$AZURE_TENANT_ID`, and `$AZURE_CLIENT_SECRET` should be configured in the operating environment of BR and TiKV.
```
Comment (en-jin19) [inline br/backup-and-restore-azblob.md:123]:
```suggestion
When backing up and restoring data using Azure AD, the environment variables `$AZURE_CLIENT_ID`, `$AZURE_TENANT_ID`, and `$AZURE_CLIENT_SECRET` should be configured in the operating environment of BR and TiKV.
```

File: br/backup-and-restore-storages.md [hunk 10]
```diff
@@ -99,6 +109,27 @@ When `credentials-file` is not specified, the migration tool will try to infer t
 2. Content of the file on the tool node at `~/.config/gcloud/application_default_credentials.json`
 3. When running in GCE or GAE, the credentials fetched from the metadata server.
 
+### Azblob URL parameters
+
+| URL parameter | Description |
+|:----------|:-----|
+| `account-name` | The account name of the storage |
+| `account-key` | The access key|
+| `access-tier` | Storage class of the uploaded objects (for example, `Hot`, `Cool`, `Archive`). If `access-tier` is not set (the value is empty), the value is `Hot` by default. |
```
Comment (en-jin19) [inline br/backup-and-restore-storages.md:118]:
```suggestion
| `access-tier` | Access tier of the uploaded objects (for example, `Hot`, `Cool`, `Archive`). If `access-tier` is not set (the value is empty), the value is `Hot` by default. |
```

File: br/backup-and-restore-storages.md (hunk 2 above)
Comment (en-jin19) [inline br/backup-and-restore-storages.md:172]:
```suggestion
| `--azblob.access-tier` | Access tier of the uploaded objects (for example, `Hot`, `Cool`, `Archive`). If `access-tier` is not set (the value is empty), the value is `Hot` by default. |
```

**Review by Leavrth — APPROVED** [review]

File: br/backup-and-restore-azblob.md [hunk 11]
```diff
@@ -0,0 +1,163 @@
...
+With BR, you can back up and restore data on Azure Blob Storage in the following two methods:
+
+- Backup and restore using Azure AD
+- Backup and restore using an access key
+
+In common cases, to avoid exposing the key information (such as `account-key`) in command lines, it is recommended to use the first method - use Azure AD.
+
+The following is an example of backup and restore operations on Azure Blob Storage using the above two methods. The purpose of the operations are as follows:
+
+- Back up: Back up the database named `test` to a place with container name `container=test` and path prefix `t1` in Azure Blob Storage.
```
Comment (ran-huang) [inline br/backup-and-restore-azblob.md:27]:
```suggestion
- Back up: Back up the `test` database to a space in the `container=test` container with `t1` as the path prefix in Azure Blob Storage.
```

File: br/backup-and-restore-azblob.md [hunk 12]
```diff
@@ -0,0 +1,163 @@
...
+- Restore: Restore the place with the container name `container=test` and the path prefix `t1` in Azure Blob Storage to the database named `test`.
```
Comment (ran-huang) [inline br/backup-and-restore-azblob.md:28]:
```suggestion
- Restore: Restore data from a space in the `container=test` container with `t1` as the path prefix in Azure Blob Storage to the `test` database.
```

File: br/backup-and-restore-azblob.md [hunk 13]
```diff
@@ -0,0 +1,163 @@
...
+
+### Method 1: Back up and restore using Azure AD (recommended)
+
+In the operating environment of BR and TiKV, the environment variables `$AZURE_CLIENT_ID`, `$AZURE_TENANT_ID`, and `$AZURE_CLIENT_SECRET` should be configured. When these variables are configured, BR can use Azure AD to access Azure Blob Storage without configuring `account-key`. This method is safer and therefore recommended. `$AZURE_CLIENT_ID`, `$AZURE_TENANT_ID`, and `$AZURE_CLIENT_SECRET` respectively refer the application ID `client_id`, the tenant ID `tenant_id`, and the client password `client_secret` of Azure application.
```
Comment (ran-huang) [inline br/backup-and-restore-azblob.md:32]:
```suggestion
In the operating environment of BR and TiKV, the environment variables `$AZURE_CLIENT_ID`, `$AZURE_TENANT_ID`, and `$AZURE_CLIENT_SECRET` must be configured. When these variables are configured, BR can use Azure AD to access Azure Blob Storage without configuring `account-key`. This method is safer and therefore recommended. `$AZURE_CLIENT_ID`, `$AZURE_TENANT_ID`, and `$AZURE_CLIENT_SECRET` refer to the application ID `client_id`, the tenant ID `tenant_id`, and the client password `client_secret` of Azure application.
```

File: br/backup-and-restore-azblob.md [hunk 14]
```diff
@@ -0,0 +1,163 @@
...
+
+When backing up data using an access key, you need to specify `account-name`, `account-key`, and `access-tier`. If `access-tier` is not set (the value is empty), the value is `Hot` by default.
+
+> **Note:**
+>
+> When using Azure Blob Storage as the external storage, you must set `send-credentials-to-tikv = true` (which is set by default). Otherwise, the backup task will fail.
+
+This section shows backing up data to `cool tier`, that is, the access tier of the uploaded object is `Cool`. You can specify `account-name`, `account-key`, and `access-tier` in two ways:
+
+- Write the parameters information in URL parameters:
```
Comment (ran-huang) [inline br/backup-and-restore-azblob.md:86]:
```suggestion
- Write the parameter information in URL parameters:
```

File: br/backup-and-restore-azblob.md [hunk 15]
```diff
@@ -0,0 +1,163 @@
...
+
+    ```
+    tiup br backup db --db test -u 127.0.0.1:2379 -s 'azure://test/t1?account-name=devstoreaccount1&account-key=Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==&access-tier=Cool'
+    ```
+
+- Write the parameters information in command-line parameters:
```
Comment (ran-huang) [inline br/backup-and-restore-azblob.md:92]:
```suggestion
- Write the parameter information in command-line parameters:
```

File: br/backup-and-restore-azblob.md [hunk 16]
```diff
@@ -0,0 +1,163 @@
...
+
+    ```
+    tiup br backup db --db test -u 127.0.0.1:2379 -s 'azure://test/t1?' --azblob.account-name=devstoreaccount1 --azblob.access-tier=Cool
+    ```
+
+#### Restore
+
+When restoring data using Azure AD, you need to specify `account-name`. You can specify it in two ways:
+
+- Write the parameters information in URL parameters:
```
Comment (ran-huang) [inline br/backup-and-restore-azblob.md:62]:
```suggestion
- Write the parameter information in URL parameters:
```

File: br/backup-and-restore-azblob.md [hunk 17]
```diff
@@ -0,0 +1,163 @@
...
+
+    ```
+    tiup br restore db --db test -u 127.0.0.1:2379 -s 'azure://test/t1?account-name=devstoreaccount1'
+    ```
+
+- Write the parameters information in command-line parameters:
```
Comment (ran-huang) [inline br/backup-and-restore-azblob.md:68]:
```suggestion
- Write the parameter information in command-line parameters:
```

File: br/backup-and-restore-azblob.md [hunk 18]
```diff
@@ -0,0 +1,163 @@
...
+
+    ```
+    tiup br restore db --db test -u 127.0.0.1:2379 -s 'azure://test/t1?' --azblob.account-name=devstoreaccount1 --azblob.account-key=Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==
+    ```
+
+## Parameter specification
+
+During the backup and restore process, you need to use parameters `account-name`, `account-key`, `access-tier`. The following is the detailed description of the parameters:
```
Comment (ran-huang) [inline br/backup-and-restore-azblob.md:116]:
```suggestion
During the backup and restore process, you need to use `account-name`, `account-key`, and `access-tier`. The following is the detailed description of the parameters:
```

File: br/backup-and-restore-azblob.md [hunk 19]
```diff
@@ -0,0 +1,163 @@
...
+    tiup br restore db --db test -u 127.0.0.1:2379 -s 'azure://test/t1?account-name=devstoreaccount1&account-key=Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw=='
+    ```
+
+- Write the parameters information in command-line parameters:
+
+    ```
+    tiup br restore db --db test -u 127.0.0.1:2379 -s 'azure://test/t1?' --azblob.account-name=devstoreaccount1 --azblob.account-key=Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==
+    ```
+
+## Parameter specification
```
Comment (ran-huang) [inline br/backup-and-restore-azblob.md:114]:
```suggestion
## Parameter description
```

File: br/backup-and-restore-azblob.md [hunk 20]
```diff
@@ -0,0 +1,163 @@
...
+
+During the backup and restore process, you need to use parameters `account-name`, `account-key`, `access-tier`. The following is the detailed description of the parameters:
+
+- [URL parameters](/br/backup-and-restore-storages.md#azblob-url-parameters)
+- [Command-line parameters](/br/backup-and-restore-storages.md#azblob-command-line-parameters)
+
+### Configure environment variables as parameters
+
+When backing up and restoring data using Azure AD, the environment variables `$AZURE_CLIENT_ID`, `$AZURE_TENANT_ID`, and `$AZURE_CLIENT_SECRET` should be configured in the operating environment of BR and TiKV.
```
Comment (ran-huang) [inline br/backup-and-restore-azblob.md:123]:
```suggestion
When backing up and restoring data using Azure AD, the environment variables `$AZURE_CLIENT_ID`, `$AZURE_TENANT_ID`, and `$AZURE_CLIENT_SECRET` must be configured in the operating environment of BR and TiKV.
```

File: br/backup-and-restore-azblob.md [hunk 21]
```diff
@@ -0,0 +1,163 @@
...
+
+- When using a cluster started with TiUP, TiKV uses the "systemd" service. The following example provides how to configure the above three environment variables as parameters for TiKV:
```
Comment (ran-huang) [inline br/backup-and-restore-azblob.md:125]:
```suggestion
- When you start a cluster using TiUP, TiKV uses the systemd service. The following example provides how to configure the above three environment variables as parameters for TiKV:
```

File: br/backup-and-restore-azblob.md [hunk 22]
```diff
@@ -0,0 +1,163 @@
...
+
+    > **Note:**
+    >
+    > You need to restart TiKV in step 3. If your TiKV is not suitable to be restarted, you can back up and restore data using the [Method 2](method2back-up-and-restore-using-an-access-key-easy).
```
Comment (ran-huang) [inline br/backup-and-restore-azblob.md:129]:
```suggestion
    > You need to restart TiKV in Step 3. If your TiKV cannot be restarted, you can back up and restore data using the [Method 2](method2back-up-and-restore-using-an-access-key-easy).
```

File: br/backup-and-restore-azblob.md [hunk 23]
```diff
@@ -0,0 +1,163 @@
...
+        ```
+
+    3. Reload the configuration and restart TiKV:
+
+        ```
+        systemctl daemon-reload
+        systemctl restart tikv-24000
+        ```
+
+- For TiKV and BR started with command lines, to configure the Azure AD information to them, you only need to check whether the environment variables `$AZURE_CLIENT_ID`, `$AZURE_TENANT_ID`, and `$AZURE_CLIENT_SECRET` are configured in the operating environment. You can check whether the variables are in the operating environment of BR and TiKV by running the following command lines:
```
Comment (ran-huang) [inline br/backup-and-restore-azblob.md:153]:
```suggestion
- For TiKV and BR started with command lines, to configure the Azure AD information for them, you only need to check whether the environment variables `$AZURE_CLIENT_ID`, `$AZURE_TENANT_ID`, and `$AZURE_CLIENT_SECRET` are configured in the operating environment. You can check whether the variables are in the operating environment of BR and TiKV by running the following commands:
```

File: br/backup-and-restore-azblob.md [hunk 24]
```diff
@@ -0,0 +1,163 @@
...
+
+    ```
```
Comment (ran-huang) [inline br/backup-and-restore-azblob.md:155]:
```suggestion
    ```shell
```

File: br/backup-and-restore-storages.md [hunk 25]
```diff
@@ -99,6 +109,27 @@ When `credentials-file` is not specified, the migration tool will try to infer t
...
 
+### Azblob URL parameters
+
+| URL parameter | Description |
+|:----------|:-----|
+| `account-name` | The account name of the storage |
+| `account-key` | The access key|
+| `access-tier` | Access tier of the uploaded objects (for example, `Hot`, `Cool`, `Archive`). If `access-tier` is not set (the value is empty), the value is `Hot` by default. |
+
+To ensure that TiKV and the migration tool use the same storage account, the migration tool determines the value of `account-name`. That is, `send-credentials-to-tikv = true` is set by default. The migration tool infers these keys from the environment in the following order:
```
Comment (ran-huang) [inline br/backup-and-restore-storages.md:120]: "Migration tool" -> BR

File: br/backup-and-restore-storages.md [hunk 26]
```diff
@@ -99,6 +109,27 @@ When `credentials-file` is not specified, the migration tool will try to infer t
...
+
+1. If both `account-name` **and** `account-key` are specified, the key specified by this parameter is used.
+2. If `account-key` is not specified, then the tool tries to read the related credentials from environment variables on the node of the tool.
```
Comment (ran-huang) [inline br/backup-and-restore-storages.md:123]: the tool -> BR

File: br/backup-and-restore-storages.md [hunk 27]
```diff
@@ -99,6 +109,27 @@ When `credentials-file` is not specified, the migration tool will try to infer t
...
+    - The migration tool reads `$AZURE_CLIENT_ID`, `$AZURE_TENANT_ID`, and `$AZURE_CLIENT_SECRET` first. At the same time, the tool allows TiKV to read the above three environment variables from the respective nodes and access using Azure AD (Azure Active Directory).
+        - `$AZURE_CLIENT_ID`, `$AZURE_TENANT_ID`, and `$AZURE_CLIENT_SECRET` respectively refer the application ID `client_id`, the tenant ID `tenant_id`, and the client password `client_secret` of Azure application.
```
Comment (ran-huang) [inline br/backup-and-restore-storages.md:125]:
```suggestion
        - `$AZURE_CLIENT_ID`, `$AZURE_TENANT_ID`, and `$AZURE_CLIENT_SECRET` respectively refer to the application ID `client_id`, the tenant ID `tenant_id`, and the client password `client_secret` of Azure application.
```

File: br/backup-and-restore-storages.md (hunk 25 above)
Comment (en-jin19) [inline br/backup-and-restore-storages.md:120]:
```suggestion
To ensure that TiKV and BR use the same storage account, BR determines the value of `account-name`. That is, `send-credentials-to-tikv = true` is set by default. BR infers these keys from the environment in the following order:
```

File: br/backup-and-restore-storages.md (hunk 26 above)
Comment (en-jin19) [inline br/backup-and-restore-storages.md:123]:
```suggestion
2. If `account-key` is not specified, then BR tries to read the related credentials from environment variables on the node of BR.
```

File: br/backup-and-restore-storages.md [hunk 28]
```diff
@@ -99,6 +109,27 @@ When `credentials-file` is not specified, the migration tool will try to infer t
...
+|:----------|:-----|
+| `account-name` | The account name of the storage |
+| `account-key` | The access key|
+| `access-tier` | Access tier of the uploaded objects (for example, `Hot`, `Cool`, `Archive`). If `access-tier` is not set (the value is empty), the value is `Hot` by default. |
+
+To ensure that TiKV and BR use the same storage account, BR determines the value of `account-name`. That is, `send-credentials-to-tikv = true` is set by default. BR infers these keys from the environment in the following order:
+
+1. If both `account-name` **and** `account-key` are specified, the key specified by this parameter is used.
+2. If `account-key` is not specified, then BR tries to read the related credentials from environment variables on the node of BR.
+    - The migration tool reads `$AZURE_CLIENT_ID`, `$AZURE_TENANT_ID`, and `$AZURE_CLIENT_SECRET` first. At the same time, the tool allows TiKV to read the above three environment variables from the respective nodes and access using Azure AD (Azure Active Directory).
```
Comment (ran-huang) [inline br/backup-and-restore-storages.md:124]:
```suggestion
    - BR reads `$AZURE_CLIENT_ID`, `$AZURE_TENANT_ID`, and `$AZURE_CLIENT_SECRET` first. At the same time, BR allows TiKV to read the above three environment variables from the respective nodes and access using Azure AD (Azure Active Directory).
```

File: br/backup-and-restore-storages.md [hunk 29]
```diff
@@ -99,6 +109,27 @@ When `credentials-file` is not specified, the migration tool will try to infer t
...
+        - `$AZURE_CLIENT_ID`, `$AZURE_TENANT_ID`, and `$AZURE_CLIENT_SECRET` respectively refer to the application ID `client_id`, the tenant ID `tenant_id`, and the client password `client_secret` of Azure application.
+        - To learn how to check whether the operating system have configured `$AZURE_CLIENT_ID`, `$AZURE_TENANT_ID`, and `$AZURE_CLIENT_SECRET`, or you need to configure these variables as parameters, refer to [Configure environment variables as parameters](/br/backup-and-restore-azblob.md#configure-environment-variables-as-parameters).
```
Comment (ran-huang) [inline br/backup-and-restore-storages.md:126]:
```suggestion
        - To learn how to check whether the operating system have configured `$AZURE_CLIENT_ID`, `$AZURE_TENANT_ID`, and `$AZURE_CLIENT_SECRET`, or if you need to configure these variables as parameters, refer to [Configure environment variables as parameters](/br/backup-and-restore-azblob.md#configure-environment-variables-as-parameters).
```

File: br/backup-and-restore-storages.md [hunk 30]
```diff
@@ -99,6 +109,27 @@ When `credentials-file` is not specified, the migration tool will try to infer t
...
+3. If the above three environment variables are not configured in the node of the tool, then the tool tries to read `$AZURE_STORAGE_KEY` and access using a key.
```
Comment (ran-huang) [inline br/backup-and-restore-storages.md:127]:
```suggestion
3. If the above three environment variables are not configured in the BR node, BR tries to read `$AZURE_STORAGE_KEY` using an access key.
```

File: br/backup-and-restore-storages.md [hunk 31]
```diff
@@ -128,11 +159,18 @@ If you have specified URL parameters and command-line parameters at the same tim
...
+### Azblob command-line parameters
+
+| Command-line parameter | Description |
+| `--azblob.account-name` | The account name of the storage |
+| `--azblob.account-key` | The access key |
+| `--azblob.access-tier` | Access tier of the uploaded objects (for example, `Hot`, `Cool`, `Archive`). If `access-tier` is not set (the value is empty), the value is `Hot` by default. |
+
 ## BR sending credentials to TiKV
 
 By default, when using S3 and GCS destinations, BR will send the credentials to every TiKV nodes to reduce setup complexity.
```
Comment (ran-huang) [inline br/backup-and-restore-storages.md:176]:
```suggestion
By default, when using S3, GCS, or Azblob destinations, BR will send the credentials to every TiKV node to reduce setup complexity.
```

File: br/backup-and-restore-azblob.md [hunk 32]
```diff
@@ -0,0 +1,163 @@
...
+
+### Configure environment variables as parameters
+
+When backing up and restoring data using Azure AD, the environment variables `$AZURE_CLIENT_ID`, `$AZURE_TENANT_ID`, and `$AZURE_CLIENT_SECRET` must be configured in the operating environment of BR and TiKV.
+
+- When you start a cluster using TiUP, TiKV uses the systemd service. The following example provides how to configure the above three environment variables as parameters for TiKV:
+
+    > **Note:**
+    >
+    > You need to restart TiKV in Step 3. If your TiKV cannot be restarted, you can back up and restore data using the [Method 2](method2back-up-and-restore-using-an-access-key-easy).
```
Comment (en-jin19) [inline br/backup-and-restore-azblob.md:129]:
```suggestion
    > You need to restart TiKV in Step 3. If your TiKV cannot be restarted, you can back up and restore data using the [Method 2](#method2back-up-and-restore-using-an-access-key-easy).
```

File: br/backup-and-restore-azblob.md [hunk 33]
```diff
@@ -0,0 +1,163 @@
...
+
+### Configure environment variables as parameters
+
+When backing up and restoring data using Azure AD, the environment variables `$AZURE_CLIENT_ID`, `$AZURE_TENANT_ID`, and `$AZURE_CLIENT_SECRET` must be configured in the operating environment of BR and TiKV.
+
+- When you start a cluster using TiUP, TiKV uses the systemd service. The following example provides how to configure the above three environment variables as parameters for TiKV:
+
+    > **Note:**
+    >
+    > You need to restart TiKV in Step 3. If your TiKV cannot be restarted, you can back up and restore data using the [Method 2](#method2back-up-and-restore-using-an-access-key-easy).
```
Comment (en-jin19) [inline br/backup-and-restore-azblob.md:129]:
```suggestion
    > You need to restart TiKV in Step 3. If your TiKV cannot be restarted, you can back up and restore data using the [Method 2](#method-2-back-up-and-restore-using-an-access-key-easy).
```

File: br/backup-and-restore-azblob.md [hunk 34]
```diff
@@ -0,0 +1,163 @@
...
+During the backup and restore process, you need to use `account-name`, `account-key`, and `access-tier`. The following is the detailed description of the parameters:
+
+- [URL parameters](/br/backup-and-restore-storages.md#azblob-url-parameters)
+- [Command-line parameters](/br/backup-and-restore-storages.md#azblob-command-line-parameters)
+
+### Configure environment variables as parameters
+
+When backing up and restoring data using Azure AD, the environment variables `$AZURE_CLIENT_ID`, `$AZURE_TENANT_ID`, and `$AZURE_CLIENT_SECRET` must be configured in the operating environment of BR and TiKV.
+
+- When you start a cluster using TiUP, TiKV uses the systemd service. The following example provides how to configure the above three environment variables as parameters for TiKV:
```
Comment (en-jin19) [inline br/backup-and-restore-azblob.md:125]:
```suggestion
- When you start a cluster using TiUP, TiKV uses the "systems" service. The following example provides how to configure the above three environment variables as parameters for TiKV:
```

File: br/backup-and-restore-azblob.md [hunk 35]
```diff
@@ -0,0 +1,163 @@
...
+
+    > **Note:**
+    >
+    > You need to restart TiKV in Step 3. If your TiKV cannot be restarted, you can back up and restore data using the [Method 2](#method-2-back-up-and-restore-using-an-access-key-easy).
+
+    1. Suppose that the TiKV port on this node is 24000, that is, the name of the "systemd" service is "tikv-24000":
```
Comment (en-jin19) [inline br/backup-and-restore-azblob.md:131]:
```suggestion
    1. Suppose that the TiKV port on this node is `24000`, that is, the name of the "systemd" service is "tikv-24000":
```

File: br/backup-and-restore-azblob.md [hunk 36]
```diff
@@ -0,0 +1,163 @@
...
+
+## User scenario
+
+Azure virtual machines can quickly store large-scale data on Azure Blob Storage. If you are using Azure virtual machines to deploy your cluster, you can back up your data on Azure Blob Storage.
+
+## Usage
+
+With BR, you can back up and restore data on Azure Blob Storage by the following two methods:
+
+- Back up and restore data using Azure AD
```
Comment (en-jin19) [inline br/backup-and-restore-azblob.md:20]:
```suggestion
- Back up and restore data using Azure AD (Azure Active Directory)
```

File: br/backup-and-restore-storages.md [hunk 37]
```diff
@@ -99,6 +109,27 @@ When `credentials-file` is not specified, the migration tool will try to infer t
...
+
+1. If both `account-name` **and** `account-key` are specified, the key specified by this parameter is used.
+2. If `account-key` is not specified, then BR tries to read the related credentials from environment variables on the node of BR.
+    - BR reads `$AZURE_CLIENT_ID`, `$AZURE_TENANT_ID`, and `$AZURE_CLIENT_SECRET` first. At the same time, BR allows TiKV to read the above three environment variables from the respective nodes and access using Azure AD (Azure Active Directory).
+        - `$AZURE_CLIENT_ID`, `$AZURE_TENANT_ID`, and `$AZURE_CLIENT_SECRET` respectively refer to the application ID `client_id`, the tenant ID `tenant_id`, and the client password `client_secret` of Azure application.
+        - To learn how to check whether the operating system have configured `$AZURE_CLIENT_ID`, `$AZURE_TENANT_ID`, and `$AZURE_CLIENT_SECRET`, or if you need to configure these variables as parameters, refer to [Configure environment variables as parameters](/br/backup-and-restore-azblob.md#configure-environment-variables-as-parameters).
```
Comment (en-jin19) [inline br/backup-and-restore-storages.md:126]:
```suggestion
        - To learn how to check whether the operating system has configured `$AZURE_CLIENT_ID`, `$AZURE_TENANT_ID`, and `$AZURE_CLIENT_SECRET`, or if you need to configure these variables as parameters, refer to [Configure environment variables as parameters](/br/backup-and-restore-azblob.md#configure-environment-variables-as-parameters).
```

**Review by ran-huang — APPROVED** [review]

**Pull request merged**